image-scanner.sh

```diff
@@ -34,17 +107,25 @@ else
     echo "Found trivy, using trivy v$(trivy -v | cut -d ' ' -f 2)"
 fi
 
+# Start the text report
+summary_file_output="Trivy Scan Results Summary\n"
+summary_file_output+="===================\n"
+summary_file_output+="$(printf "%-40s %-10s %-10s %-10s %-10s %-10s\n" "Image" "CRITICAL" "HIGH" "MEDIUM" "LOW" "UNKNOWN")"
+summary_file_output+="\n$(printf '%.0s-' {1..90})\n"
+
 echo "Retrieving repository list ..."
 REPO_LIST=$(curl -s "https://hub.docker.com/v2/repositories/${ORG}/?page_size=100" | jq -r '.results|.[]|.name')
```

Comment:
For some of the images we don't have latest tag, so we have to handle that condition.

Reply:
That's handled while parsing the count.
```if [ $? -ne 0 ]; then
        report_text+=$(severity_count_row "$image (Failed)" "-" "-" "-" "-" "-")
        report_text+="\n"
        echo "$report_text"
        return 1
    fi
```
For the detailed report an empty html file is generated